operator/src/main/helm/coherence-operator/templates/secret.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at
+# http://oss.oracle.com/licenses/upl.
+
+{{- if .Values.logCaptureEnabled }}
```

Comment:
So this secret is not going to be created unless `logCaptureEnabled` is true?

What are the implications of that on the usage of secret references above?

Reply:
Also, shouldn't this file be renamed to `coherence-monitoring-config.yaml`? 

`secret.yaml` seems to generic -- we may have other secrets in the future...

Reply:
When logCaptureEnabled = false, there is no need for fluentd and the secret here.
We may define multiple secret in a yaml if it is necessary. I will rename it to coherence-monitoring-config.xml for now.